Specs/isImageOrImageBitmap.js

```diff
@@ -0,0 +1,22 @@
+define([
+        'Core/FeatureDetection',
+        'Core/Resource'
+    ], function(
+        FeatureDetection,
+        Resource) {
+    'use strict';
+
+    function isImageOrImageBitmap(image) {
```

Comment:
This would be better as a custom Jasmine matcher than a standlone function, I'll just submit a fix to master after merging.